pkgs/applications/misc/remnote/default.nix

```diff
@@ -29,22 +32,25 @@ in
     categories = [ "Office" ];
     mimeTypes = [ "x-scheme-handler/remnote" "x-scheme-handler/rn" ];
   };
+
   dontUnpack = true;
   dontConfigure = true;
   dontBuild = true;
+
   installPhase = ''
     runHook preInstall
 
-    install -D ${appexec}/bin/remnote-${version} $out/bin/remnote
-    install -m 444 -D "${desktopItem}/share/applications/"* -t $out/share/applications/
-    install -m 444 -D ${icon} $out/share/pixmaps/remnote.png
+    install -Dm755 ${appexec}/bin/remnote-${version} $out/bin/remnote
+    install -Dm444 "${desktopItem}/share/applications/"* -t $out/share/applications/
+    install -Dm444 ${icon} $out/share/pixmaps/remnote.png
 
     runHook postInstall
   '';
+
   meta = with lib; {
     description = "A note-taking application focused on learning and productivity";
     homepage = "https://remnote.com/";
-    maintainers = with maintainers; [ max-niederman chewblacka ];
+    maintainers = with maintainers; [ chewblacka ];
```

Comment:
Why?

Reply:
Maintainer hasn't been active on this package in at least the last 6 months. Keeps auto-adding him as a reviewer and not giving any other suggestions.

Reply:
> Why?
Maintainer hasn't been active on this package in at least ~~the last 6 months~~.

Make that in nearly 2 years (May 2022).

Reply:
Please split out this change into a separate PR.
There should be at least a distinct maintainer timeout on a removal, that isn't bound to a package update (defering the package update needlesly).

Reply:
> Please split out this change into a separate PR. There should be at least a distinct maintainer timeout on a removal, that isn't bound to a package update (defering the package update needlesly).

Done.